### ISSUE TITLE
How to evaluate model with current weights?

### ISSUE THREAD

**Issue by KacperKubara** [speechbrain/speechbrain#1507]
Hi,

While I am running a pretrained CRDNN model, I finetune it on some data with .fit() method. Then, I would like to see its performance with .evaluate() function.

However, when I run the evaluate() function the log is something like this:
```
  0%|          | 0/16 [00:00<?, ?it/s]
  6%|▋         | 1/16 [00:22<05:42, 22.84s/it]
 12%|█▎        | 2/16 [00:29<03:03, 13.12s/it]
 19%|█▉        | 3/16 [00:54<04:04, 18.84s/it]
 25%|██▌       | 4/16 [01:21<04:23, 21.98s/it]
 31%|███▏      | 5/16 [01:47<04:18, 23.45s/it]
 38%|███▊      | 6/16 [02:17<04:17, 25.71s/it]
 44%|████▍     | 7/16 [02:43<03:50, 25.65s/it]
 50%|█████     | 8/16 [03:11<03:31, 26.41s/it]
 56%|█████▋    | 9/16 [03:20<02:27, 21.08s/it]
 62%|██████▎   | 10/16 [03:26<01:37, 16.31s/it]
 69%|██████▉   | 11/16 [03:40<01:17, 15.56s/it]
 75%|███████▌  | 12/16 [04:08<01:18, 19.58s/it]
 81%|████████▏ | 13/16 [04:38<01:08, 22.67s/it]
 88%|████████▊ | 14/16 [04:50<00:38, 19.27s/it]
 94%|█████████▍| 15/16 [04:57<00:15, 15.83s/it]
100%|██████████| 16/16 [05:35<00:00, 22.38s/it]
100%|██████████| 16/16 [05:35<00:00, 20.98s/it]
speechbrain.utils.train_logger - Epoch loaded: 15 - test loss: 1.50, test CER: 31.43, test WER: 37.69
```

As you can see, it seems that model is evaluated on Epoch 15 which probably indicates that it loads weights from that epoch instead of using the weights which have just been finetuned. Whenever I do this finetuning and then evaluate(), the scores are **exactly** the same (up to 8 decimal digits). How do I overcome that?

TLDR; I want to use current weights with .evaluate() function, not with weights loaded from best epoch. How do I achieve that?

**Comment by KacperKubara** [comment]
Hi @TParcollet, do you perhaps know how to overcome that? :)

**Comment by TParcollet** [comment]
Hi @KacperKubara, yes I do. However, let me first redirect you to the corresponding [tutorial](https://colab.research.google.com/drive/1VH7U0oP3CZsUNtChJT2ewbV_q1QX8xre?usp=sharing) as you could learn much more than just this feature from our checkpointer ;-)

**Comment by KacperKubara** [comment]
Ok, nice thanks!

**Comment by KacperKubara** [comment]
Alright, but I guess my questions is then how can I turn off the checkpoint loading when I trigger the evaulate() function from Brain class?

Should i delete the checkpointer with `del brain.checkpointer`, or do `brain.checkpointer.delete_checkpoints(num_to_keep=0)` ?

**Comment by TParcollet** [comment]
I will let our Master of checkpoints @Gastron answer here. I guess he is in vacation though.

**Comment by TParcollet** [comment]
(In practice, you can, I believe, just pass an argument that forces to use the latest instead of the best using the keys).

**Comment by KacperKubara** [comment]
`brain.checkpointer=None` works.
I looked over the source code for ` brain.checkpointer.delete_checkpoints(num_to_keep=0)` but this delets the checkpoints from the directory which is not exactly what I want to do.
